### DISCUSSION TITLE
Hindi Translation

### DISCUSSION THREAD

**Discussion by vishnuchd** [vuejs-translations/guidelines#17] [Ideas]
I would like to start the Hindi translation. If anyone is interested  It would be Great. I'd be glad to contribute. (मैं हिंदी अनुवाद शुरू करना चाहूंगा। यदि किसी को दिलचस्पी है, तो यह बहुत अच्छा होगा। मुझे योगदान करने में खुशी होगी।)

**Comment by Akshit1025** [comment]
I am also willing to contribute to this repo

**Reply by thetejasmagade** [reply]
Hey akshit, I started new Discussion see here [https://github.com/vuejs/docs/discussions/1910]( https://github.com/vuejs/docs/discussions/1910)
and also forked the project.

It is a github repo, here we have to make contributions.
https://github.com/thetejasmagade/hn.vue.docs

come and join us

**Comment by Akshit1025** [comment]
I request the repository owner to delete this discussion as this is inactive on translations

Thank You
